test/inductor/test_torchinductor.py

```diff
@@ -2013,6 +2023,7 @@ def fn(a, b):
 
     @skipCUDAIf(TEST_WITH_ROCM, "associative_scan is not supported on ROCm")
     @skip_if_halide  # scan ops
+    @skip_if_dynamic  # TODO: support lifted symints when dynamic
```

Comment:
Does this PR "break" support for associative_scan with dynamic shapes?

Reply:
Yes, because associative_scan cannot handle lifted tensors/symbols right now. @bohnstingl has an on-going change for supporting it here: https://github.com/bohnstingl/pytorch/pull/5/files. We can probably forward fix it?

Reply:
If you want